packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -532,7 +532,9 @@ Possible alternatives for such technical user authentication are BasicAuthentica
       case 'on-premise':
         return addProxyConfigurationOnPrem(
           destination,
-          this.subscriberToken?.userJwt
+          this.subscriberToken?.userJwt || this.subscriberToken?.type === 'iss'
```

Comment:
```suggestion
          this.subscriberToken?.userJwt ?? this.subscriberToken?.type === 'iss'
```
[nth]

Reply:
somehow the tenary operator was behaving strange so I refactored.

Reply:
`??` is nullish coalescing (like `||` but only returns the right hand value on `null` or `undefined`). Ternary is `?`